entries/jQuery.fx.interval.xml

```diff
@@ -1,12 +1,13 @@
 <?xml version="1.0"?>
-<entry type="property" name="jQuery.fx.interval" return="Number">
+<entry type="property" name="jQuery.fx.interval" return="Number" deprecated="3.0">
```

Comment:
I see. I guess this makes sense to target 3.0 then, our Core ticket still mentions 3.2.0, though, IIRC.

I'll see what's up with other deprecations.